_data-prepper/common-use-cases/s3-logs.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: S3 logs
+parent: Common use cases
+nav_order: 20
+---
+
+# S3 logs
+
+Data Prepper supports loading logs from [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (Amazon S3), including traditional logs, JSON documents, and CSV logs.
```

Comment:
"Data Prepper allows you to load logs from ..."  or, "With Data Prepper, you can load logs from...." or "Data Prepper can..." or, Data Prepper supports the loading of logs..." @natebower?

Reply:
Switched to "allows you too".